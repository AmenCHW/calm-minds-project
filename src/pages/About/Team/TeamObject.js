import img1 from '../img/IMG_20230125_191456_491.jpg';
import img2 from '../img/istockphoto-849220498-170667a.jpg';
import img4 from '../img/img.PNG';
import img5 from '../img/sea.PNG';

const TeamObject = [
  {
    id: 1,
    src: img1,
    alt: 'img1',
    Name: 'Ameen Abdulsamad',
    title: 'Web Developer',
  },
  {
    id: 2,
    src: img2,
    alt: 'img2',
    Name: 'Danah Osta',
    title: 'Web Developer',
  },
  {
    id: 3,
    src: img5,
    alt: 'img3',
    Name: 'Huda Hamid',
    title: 'Web Developer',
  },
  {
    id: 4,
    src: img4,
    alt: 'img4',
    Name: 'Hogr Honar',
    title: 'Web Developer',
  },
];

export default TeamObject;
