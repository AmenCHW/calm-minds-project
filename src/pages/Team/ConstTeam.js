import img1 from './img/Rectangle.png';
import img2 from './img/Rectangle1.png';
import img3 from './img/Rectangle2.png';
import img4 from './img/Rectangle3.png';
import img5 from './img/Rectangle4.png';
import img6 from './img/Rectangle 5.png';

const constTeam = [
  {
    id: 1,
    src: img1,
    alt: 'img1',
    Name: 'Allan Saleh',
    title: 'Lead Engineer& Web Developer',
  },
  {
    id: 2,
    src: img2,
    alt: 'img2',
    Name: 'Payam Abubakr',
    title: 'Junior Designer& FE Developer',
  },
  {
    id: 3,
    src: img3,
    alt: 'img3',
    Name: 'Ahmed Mohammed',
    title: 'Junior Designer& FE Developer',
  },
  {
    id: 4,
    src: img4,
    alt: 'img4',
    Name: 'Abdulmajeed Jaafer',
    title: 'Junior Designer& FE Developer',
  },
  {
    id: 5,
    src: img5,
    alt: 'img5',
    Name: 'Snoor Madih',
    title: 'Junior Designer& FE Developer',
  },
  {
    id: 6,
    src: img6,
    alt: 'img6',
    Name: 'Avraz Zebary',
    title: 'Junior Designer& FE Developer',
  },
];

export default constTeam;
