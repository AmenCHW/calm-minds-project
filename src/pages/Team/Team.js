import React from "react";

function Team(){
    return(
       <h1 className="mx-auto text-center text-7xl">Team</h1>
    )
}

export default Team;