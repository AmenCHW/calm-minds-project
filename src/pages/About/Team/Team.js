import React from 'react';
import TeamObject from './TeamObject';

function Team() {
  return (
    <div className="mx-auto lg:max-w-7xl px-10 py-10">
      <h1 className="pt-5 text-3xl sm:text-4xl md:text-5xl format-normal leading-normal">
        WE ARE HEALING, NICE TO MEET YOU!
      </h1>

      <h4 className="text-[#424A4F] text-2xl md:text-3xl py-14 md:mt-32">
        Meet the Team!
      </h4>

      <div className="flex flex-wrap pb-24">
        {TeamObject.map(({ id, src, alt, Name, title }) => {
          return (
            <div
              key={id}
              className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] max-w-[160px]
            pb-28 my-4 mx-7 sm:ml-0 lg:mr-14"
            >
              <img
                className="h-full border-b-8 border-[#a3e635]"
                src={src}
                alt={alt}
              />
              <p className="text-center py-4 px-2"> {Name}</p>
              <p className="text-center px-1">{title}</p>
            </div>
          );
        })}
      </div>
    </div>
  );
}

export default Team;
