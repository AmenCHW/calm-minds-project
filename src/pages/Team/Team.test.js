import React from "react";
import {render,screen } from "@testing-library/react"
import Team from "./Team";
import renderer from 'react-test-renderer' 

test("first snapshot testing",()=>{
    const component=renderer.create( <Team/>).toJSON()  
    expect( component).toMatchSnapshot();
})


test("test healing", async() => {
    render(
       <Team title ="Healing" />
   );
 const headingElement = screen.getAllByText(/healing/i);
   expect(headingElement).toBeInTheDocument;
 });

