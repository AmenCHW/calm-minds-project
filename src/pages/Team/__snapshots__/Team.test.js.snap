// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`first snapshot testing 1`] = `
<div
  className="mx-auto lg:max-w-7xl px-10 py-10"
>
  <h1
    className="  pt-5 text-3xl md:text-5xl format-normal leading-normal"
  >
    WE ARE HEALING, NICE TO MEET YOU!
  </h1>
  <div
    className="mt-32 px-4 "
  >
    <h4
      className=" text-3xl  py-14    lg:mx-0  mx-2 pl-5 sm:ml-10 justify-center "
    >
      Meet the Team!
    </h4>
    <div
      className="flex flex-wrap px-0 pb-24   lg:justify-between   "
    >
      <div
        className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] justify-center pb-28  px-0 my-4 ml-10 lg:mx-2 sm:justify-center "
      >
        <img
          alt="img1"
          className="w-35 h-40 "
          src="Rectangle.png"
        />
        <hr
          className="border-[#a3e635] border-4 bg-[#a3e635]"
        />
        <p
          className="text-center py-4  "
        >
           
          Allan Saleh
        </p>
        <p
          className=" pl-4 max-w-[150px]  "
        >
          Lead Engineer& Web Developer
        </p>
      </div>
      <div
        className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] justify-center pb-28  px-0 my-4 ml-10 lg:mx-2 sm:justify-center "
      >
        <img
          alt="img2"
          className="w-35 h-40 "
          src="Rectangle1.png"
        />
        <hr
          className="border-[#a3e635] border-4 bg-[#a3e635]"
        />
        <p
          className="text-center py-4  "
        >
           
          Payam Abubakr
        </p>
        <p
          className=" pl-4 max-w-[150px]  "
        >
          Junior Designer& FE Developer
        </p>
      </div>
      <div
        className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] justify-center pb-28  px-0 my-4 ml-10 lg:mx-2 sm:justify-center "
      >
        <img
          alt="img3"
          className="w-35 h-40 "
          src="Rectangle2.png"
        />
        <hr
          className="border-[#a3e635] border-4 bg-[#a3e635]"
        />
        <p
          className="text-center py-4  "
        >
           
          Ahmed Mohammed
        </p>
        <p
          className=" pl-4 max-w-[150px]  "
        >
          Junior Designer& FE Developer
        </p>
      </div>
      <div
        className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] justify-center pb-28  px-0 my-4 ml-10 lg:mx-2 sm:justify-center "
      >
        <img
          alt="img4"
          className="w-35 h-40 "
          src="Rectangle3.png"
        />
        <hr
          className="border-[#a3e635] border-4 bg-[#a3e635]"
        />
        <p
          className="text-center py-4  "
        >
           
          Abdulmajeed Jaafer
        </p>
        <p
          className=" pl-4 max-w-[150px]  "
        >
          Junior Designer& FE Developer
        </p>
      </div>
      <div
        className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] justify-center pb-28  px-0 my-4 ml-10 lg:mx-2 sm:justify-center "
      >
        <img
          alt="img5"
          className="w-35 h-40 "
          src="Rectangle4.png"
        />
        <hr
          className="border-[#a3e635] border-4 bg-[#a3e635]"
        />
        <p
          className="text-center py-4  "
        >
           
          Snoor Madih
        </p>
        <p
          className=" pl-4 max-w-[150px]  "
        >
          Junior Designer& FE Developer
        </p>
      </div>
      <div
        className="flex flex-col border-solid rounded-3xl bg-[#EAF8F9] justify-center pb-28  px-0 my-4 ml-10 lg:mx-2 sm:justify-center "
      >
        <img
          alt="img6"
          className="w-35 h-40 "
          src="Rectangle 5.png"
        />
        <hr
          className="border-[#a3e635] border-4 bg-[#a3e635]"
        />
        <p
          className="text-center py-4  "
        >
           
          Avraz Zebary
        </p>
        <p
          className=" pl-4 max-w-[150px]  "
        >
          Junior Designer& FE Developer
        </p>
      </div>
    </div>
  </div>
</div>
`;
